genfragments/ThirteenPointSixTeV/JpsiEE_Pt_2_30_13p6TeV_TuneCP5_pythia8_cfi.py

```diff
@@ -0,0 +1,53 @@
+import FWCore.ParameterSet.Config as cms
+from Configuration.Generator.Pythia8CommonSettings_cfi import *
+from Configuration.Generator.MCTunes2017.PythiaCP5Settings_cfi import *
+
+generator = cms.EDFilter("Pythia8GeneratorFilter",
```

Comment:
please use concurrent mode, "Pythia8ConcurrentGeneratorFilter"

Reply:
I have made the changes and squashed the commit. Can you please tell me if this will create a big difference in simulation because I was already running a private MC production? I am still at the GEN stage. I can redo the GEN step now instead of realizing that there is a significant change later.